src/Stryker.Core/Stryker.Core/Mutators/DefaultParameterMutator.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+using Stryker.Core.Options;
+
+namespace Stryker.Core.Mutators;
+
+public class DefaultParameterMutator : MutatorBase<InvocationExpressionSyntax>
+{
+    private readonly ICsharpMutantOrchestrator _orchestrator;
+    private readonly StrykerOptions _options;
+
+    public DefaultParameterMutator(ICsharpMutantOrchestrator orchestrator, StrykerOptions options)
+    {
+        _orchestrator = orchestrator;
+        _options = options;
+    }
+
+    public override IEnumerable<Mutation> ApplyMutations(InvocationExpressionSyntax node, SemanticModel semanticModel)
+    {
+        var methodSymbol = (IMethodSymbol)semanticModel.GetSymbolInfo(node).Symbol;
+        if (methodSymbol is null)
```

Comment:
I think we should check whether the symbol is internal or external. We might not want to mutate optional parameters of external APIs like `System.Linq`. @rouke-broersma what do you think?